proxy/src/schema.rs

```diff
@@ -1,18 +1,59 @@
-use juniper::{FieldError, FieldResult, ParseScalarResult, ParseScalarValue, RootNode, Value};
-use juniper::{FieldResult, ParseScalarResult, ParseScalarValue, RootNode, Value};
+use juniper::{
+    FieldError, IntoFieldError, ParseScalarResult, ParseScalarValue, RootNode, Value,
+};
 use std::collections::hash_map::DefaultHasher;
 use std::hash::{Hash, Hasher};
 use std::sync::Arc;
-use substrate_subxt::error::Error as SubstrateError;
 
-use radicle_registry_client::Error as RegistryError;
 use radicle_surf::{
     file_system::{Path, SystemType},
     git::{git2, BranchName, GitBrowser, GitRepository, Sha1, TagName},
 };
 
 use crate::source::{AccountId, Project, ProjectId, Source};
 
+enum Error {
+    GitError(radicle_surf::git::GitError),
+    String(String)
```

Comment:
`String` is a bit of a weird error name 🤔 Maybe just use the name `Proxy` if they're specific to the application side of things.

Reply:
Agreed with that, we can use something like `CatchAll` for the time being, as this is intended to be temporary.

Reply:
`CatchAll` applied.